docs/src/modules/components/DemoEditor.tsx

```diff
@@ -20,14 +20,17 @@ const StyledMarkdownElement = styled(MarkdownElement)(({ theme }) => [
       border: `1px solid ${(theme.vars || theme).palette.divider}`,
       colorScheme: 'dark',
       '&:hover': {
-        boxShadow: `0 0 0 3px ${(theme.vars || theme).palette.primary.light}`,
+        boxShadow: `0 0 0 3px ${alpha((theme.vars || theme).palette.primary[500], 0.5)}`,
       },
       '&:focus-within': {
-        boxShadow: `0 0 0 2px ${(theme.vars || theme).palette.primary.main}`,
+        boxShadow: `0 0 0 3px ${alpha((theme.vars || theme).palette.primary[500], 0.8)}`,
       },
       [theme.breakpoints.up('sm')]: {
         borderRadius: '0 0 12px 12px',
       },
+      ...theme.applyDarkStyles({
+        borderTop: 1,
+      }),
```

Comment:
A bit weird. Why the border depends on the light/dark mode?

Reply:
I think that's on me 😬 I'll take a closer look to get to the root, as I remember an extra border was rendered just in dark mode. And given we use opacity for the border color, they looked particularly brighter due to the colors adding up.